services/gcp/cloudsql/sslCerts.yml

```diff
@@ -0,0 +1,32 @@
+name: Google Cloud SQL
+description: >-
+  SSL client certificates for connecting to a Cloud SQL instance. Each instance can have 10 client certificates.
+scope: CRITICAL
+notes: >-
+  Cloud SQL is used to store and serve sensitive and application-critical data. Breach of a cloud SQL database can lead to exfiltration of highly sensitive data, or interruption of mission-critical applications.
+privileges:
+  create:
+    risks: [impact:consumption, escalation:network]
+    notes: >-
+      By creating a client SSL certificate for the instance, they will be able to establish a connection to the instance.
```

Comment:
who is "they"?

Reply:
the user with the certificate, I reworded this